c/src/proactor/epoll-internal.h

```diff
@@ -360,6 +384,13 @@ pcontext_t *pni_raw_connection_context(praw_connection_t *rc);
 praw_connection_t *pni_batch_raw_connection(pn_event_batch_t* batch);
 void pni_raw_connection_done(praw_connection_t *rc);
 
+bool ctimerq_init(pn_proactor_t *p);
+void ctimerq_finalize(pn_proactor_t *p);
+bool ctimerq_register(connection_timerq_t *ctq, pconnection_t *pc);
+void ctimerq_deregister(connection_timerq_t *ctq, pconnection_t *pc);
+void ctimerq_schedule_tick(connection_timerq_t *ctq, pconnection_t *pc, uint64_t deadline, uint64_t now);
+pn_event_batch_t *ctimerq_process(connection_timerq_t *ctq, bool timeout);
+
```

Comment:
These are misnamed all internal API has pni_ prefix

Also ctimerq_t is not the struct name so they are misnamed that that way too. 
If connection_timerq_t is too long in one context it is in all contexts! I don't see the relevance of the connection_ bit. Why not just timer_queue_t? Why is it specific to connections? Shouldn't it also handle the proactor timers too?